python/chronos/test/bigdl/chronos/autots/test_init.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import pytest
+from unittest import TestCase
+import os
+
+class TestLD(TestCase):
+    def test_ld_preload(self):
+        try:
+            LD = os.environ["LD_PRELOAD"]
+        except:
+            pass
```

Comment:
we may need to
1. set LD_PRELOAD
2. import autots
3. check if it has been successfully unset